examples/inverse/mixed_source_space_inverse.py

```diff
@@ -26,16 +25,16 @@
 # Set dir
 data_path = mne.datasets.sample.data_path()
 subject = 'sample'
-data_dir = op.join(data_path, 'MEG', subject)
-subjects_dir = op.join(data_path, 'subjects')
-bem_dir = op.join(subjects_dir, subject, 'bem')
+data_dir = (data_path / 'MEG' / subject)
+subjects_dir = (data_path / 'subjects')
+bem_dir = (subjects_dir / subject / 'bem')
```

Comment:
You don't need the parenthesis to define a path, `data_dir = data_path / 'MEG' / subject` is sufficient.

Reply:
Thanks, I can go in and change that. I'm not sure what you mean by change to draft?

Reply:
I'm guessing this was a comment from someone else. A PR can be in different states: Open (default), Draft, Closed (Not merged), or Closed (merged) which is denoted by different PR symbols and colors. In the top right corner, below reviewers, you should have `Still in progress? Convert to draft`. If you click on it, it will set the state to Draft. 
Alternatively, as you did here, adding the `WIP` keyword (Work In Progress) in the title indicates to other contributors that your PR is still a draft and is not yet ready for review/merge. When you are done with your modifications, you can change the state again to an open PR, and/or you can change the WIP keyword with MRG to indicate that the PR is ready.